lib/stop_times.js

```diff
@@ -205,7 +205,11 @@ WITH stop_times_based AS NOT MATERIALIZED (
 			+ departure_time
 		) t_departure,
 		s.stop_id, stops.stop_name,
-		stations.stop_id station_id, stations.stop_name station_name
+		stations.stop_id station_id, stations.stop_name station_name,
+		CASE WHEN stops.wheelchair_boarding IS NULL OR stops.wheelchair_boarding = 
```

Comment:
Not sure if you find `coalesce()` more understandable, but we could write whole the expression as follows. What do you think?

```sql
		CASE WHEN
			stops.wheelchair_boarding IS NULL
			OR stops.wheelchair_boarding = 'no_info_or_inherit'
		THEN
			coalesce(stations.wheelchair_boarding, 'no_info_or_inherit')
		ELSE
			stops.wheelchair_boarding
		END AS wheelchair_boarding
```

Reply:
Thanks! Coalesce would make this more readable

Reply:
Would you also mind indenting your `CASE` expressions like (or similarly, I don't care about the exact indentation) I did it?

In `gtfs-via-postgres`, I keep the SQL statements this way to help with readability, and diff-ability.